profiles/package.mask

```diff
@@ -1,3 +1,11 @@
+# P.B. Lecavalier <pbl.ltx@gmail.com> (20 Nov 2019)
+# Packages required for stack-2.1.3.1, which is masked,
+# and in turn require masked packages:
+# pantry needs >=dev-haskell/hpack-0.31.2
```

Comment:
Are there any issues with bumping `hpack` up to 0.31.2? I'm happy to do this unless there are reasons not to.

Reply:
I don't know if it actually works with 0.31.2, but as nothing indicates that it would not, let's go for that bump.

Reply:
Problem. Downgrading to 0.31.2: Test phase for hpack-0.31.2 fails.

Complete test log [there](http://dpaste.com/0SH5KE8).

Note that this is attempted with a bunch of masks lifted to build latest hlint and stack. Maybe results could vary with a strict application of package.mask as it stands.